bin/ch/ch.cpp

```diff
@@ -310,6 +310,12 @@ HRESULT RunScript(const char* fileName, LPCSTR fileContents, BYTE *bcBuffer, cha
             }
 
             runScript = ChakraRTInterface::JsTTDRunScript(-1, fileContents, WScriptJsrt::GetNextSourceContext(), fullPath, nullptr /*result*/);
+
+            if (runScript == JsErrorCategoryUsage)
+            {
+                wprintf(_u("FATAL ERROR: Core was built without ENABLE_TTD.\n"));
```

Comment:
That's the problem here. `ENABLE_TTD` can be triggered by the consumer. So, in that case abort and show this message because it is hard to process later.